ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -51,6 +51,15 @@ tests:
     make lint
   container:
     from: golangci-lint
+- as: unit
+  commands: make test
+  container:
+    from: src
```

Comment:
How does that works? I don't see this image ref in `images:`.

Reply:
This container is short for `pipeline:src`, which clones the source code and uses the `pipeline:root` as a base image, which is short for the `build_root`, defined a few lines above this block.